src/components/Mdx/code.tsx

```
@@ -24,21 +24,24 @@ export const Code: React.FC<unknown> = ({ children }) => {
                         <span className="red"></span>
                         <span className="yellow"></span>
                         <span className="green"></span>
+                        <div
+                            className="cursor-pointer max-sm:hidden ml-auto flex items-center"
+                            onClick={copyToClipboard}
+                        >
+                            {isCopied ? (
```

ESLint

[error] Expected newline between test and consequent of ternary expression.

(style/multiline-ternary)